Gemfile

```diff
@@ -6,4 +6,11 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 # Declare your gem's dependencies in topological_inventory.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
+gem 'rbac-api-client', :git => "https://github.com/RedHatInsights/insights-rbac-api-client-ruby", :branch => "master"
+
+group :development, :test do
+  gem 'climate_control'
```

Comment:
I don't understand the point of adding this gem.  Would rather we could avoid this.

Reply:
This gem is used in rspec file to set env variable for testing

Reply:
right, but you can also just use existing RSpec stuff to do that.